include/flashinfer/trtllm/fmha/fmhaKernels.cuh

```diff
@@ -213,15 +234,26 @@ class TllmGenFmhaKernel {
   inline bool isMlaGenKernel(RunnerParams const& params) const {
     return params.mHeadDimQk == 576 && params.mHeadDimV == 512;
   }
-  // Compute the number of CTAs in X, Y and Z dimension.
-  std::tuple<int, int, int> computeNumCtas(RunnerParams const& params, KernelMeta const& kernelMeta,
-                                           SelectKernelParams& selectKernelParams) const {
+
+  // Compute the number of CTAs in X, Y and Z dimension and the cluster size in the X dimension.
+  using CtaInfo = std::tuple<int, int, int, int, int, int>;
+
+  CtaInfo computeNumCtas(RunnerParams const& params, KernelMeta const& kernelMeta,
```

Comment:
Better to rename it `computeCtaAndClusterConfig` because it returns not only cta information.